package.json

```diff
@@ -9,8 +9,9 @@
   "main": "index.html",
   "scripts": {
     "build:icons": "node ./cli/icon-generator.js",
+    "build:lang": "node ./cli/lang-generator.js",
     "lint": "eslint . --ext .js,.html",
-    "postinstall": "npm run build:icons",
+    "postinstall": "npm run build:icons && npm run build:lang",
```

Comment:
I think it would generally be best to publish the generated files, and not overwrite them unless `build:lang` is explicitly run.  The chance of them being different from one install to the next is minimal, since there aren't really any dependencies other than `fs` and `path`.  Nevertheless, I think we could avoid the overwrite scenario for consumers.

Reply:
Just like the icons, these will be in the published NPM package once I have that working later today or tomorrow. So there won't be a need to generate them post-install like this -- I'll remove that step then though.

Reply:
Removed it and updated the `build:icons` to not delete all the generated files.